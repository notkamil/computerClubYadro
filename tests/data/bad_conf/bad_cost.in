3
08:00 20:00
something very bad again piff paff kek lol 25345234