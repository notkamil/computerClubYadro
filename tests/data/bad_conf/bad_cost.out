something very bad again piff paff kek lol 25345234